soh/soh/Enhancements/randomizer/3drando/location_access/locacc_ice_cavern.cpp

```diff
@@ -53,24 +53,24 @@ void AreaTable_Init_IceCavern() {
   }, {}, {
                   //Exits
                   Entrance(ICE_CAVERN_ENTRYWAY,             {[]{return true;}}),
-                  Entrance(ICE_CAVERN_MQ_MAP_ROOM,          {[]{return IsAdult || CanUse(DINS_FIRE) || (HasExplosives && (CanUse(STICKS) || CanUse(SLINGSHOT) || KokiriSword));}}),
+                  Entrance(ICE_CAVERN_MQ_MAP_ROOM,          {[]{return CanUse(MASTER_SWORD) || CanUse(BIGGORON_SWORD) || CanUse(DINS_FIRE) || (HasExplosives && (CanUse(STICKS) || CanUse(SLINGSHOT) || CanJumpslash));}}),
```

Comment:
```suggestion
                  Entrance(ICE_CAVERN_MQ_MAP_ROOM,          {[]{return CanUse(MASTER_SWORD) || CanUse(KOKIRI_SWORD) || CanUse(BIGGORON_SWORD) || CanUse(DINS_FIRE) || (HasExplosives && (CanUse(STICKS) || CanUse(SLINGSHOT) || CanUse(BOW)));}}),
```

CanJumpslash would only be checking for kokiri sword which ive moved to earlier anyway
I've put Bow here too but basing it off my memory right now